pallets/chain-space/src/lib.rs

```diff
@@ -915,6 +967,138 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Creates a new space with a unique identifier based on the provided
```

Comment:
Are the comments auto-generated or defined?

Reply:
Comments are written. There is a tool / extension in few editors to write comments, you can use them.